lib/tasks/emotion_selection_notification.rake

```diff
@@ -3,4 +3,9 @@ namespace :notifications do
   task emotion_selection_invite_all: :environment do
     EmotionSelectionNotificationWorker.new.run_notification
   end
+
+  desc 'Send results email'
+  task send_results_email: :environment do
+    EmotionSelectionNotificationWorker.new.run_notification(:send_results_email)
```

Comment:
@LyubovP it seems that it would be better to move your code in other worker with proper name smth like `ResultEmailNotificationWorker`. It will be clearer and allow to remove unnecessary complexity in the worker itself